lib/json_api_controller/json_schema_validator.rb

```diff
@@ -7,7 +7,8 @@ module JsonSchemaValidator
 
     included do
       @action_params = Hash.new
-      (@actions & %i(update create)).each do |action|
+      schema_actions = %i(update create) | Array.wrap(@extra_schema_actions)
+      (@actions | schema_actions).each do |action|
```

Comment:
don't change this to be the set union it is the set intersection on purpose. 
`(@actions & schema_actions)`